manual/azure-sru-19.2.21.txt

```diff
@@ -1,14 +1,22 @@
+#!/bin/bash
 # Manually deploy on Azure using Azure CLI client.
 # Validate upgrade to proposed version of cloud-init
 # Check Tracebacks or error conditions on clean boot
- 
+
 # To be adapted to the SRU to test
 SRU_SERIES="xenial bionic disco"
 
+# local values
```

Comment:
Good additions, can you also add this content to the sru-templates/manual/azure-sru file so it gets included next SRU?

Reply:
Yes!  I'll put up a new PR for these changes.